### ISSUE TITLE
cannot connect to mysql running in docker container

### ISSUE THREAD

**Issue by mamartins** [sequelpro/sequelpro#2871]
every time I to connect sequel pro a to a mysql running instance inside a docker container, after I select a database the app crashes.

**Comment by abhibeckert** [comment]
What version of sequel pro? This may be fixed in the next release.

Please try downloading the latest test build to verify: https://sequelpro.com/test-builds

**Comment by mamartins** [comment]
it happens with version 1.1.2. It works with latest test build

**Comment by dmoagx** [comment]
Duplicate of #2606

**Comment by AryanJ-NYC** [comment]
This happens for me, too. Sequel Pro 1.1.12.

**Comment by levonbarker** [comment]
Having the same issue.

**Comment by GroomedGorilla** [comment]
Having the same issue, but the test-builds fixed this. Working with that for now (Snapshot build 5224), but any chance this is coming to Sequel Pro stable soon?

**Comment by mwendell** [comment]
After a few hours of futzing with it, I was able to get this working using the latest nightly build (5438), but only after I updated my user to use **mysql_native_password**. I was frustrated because MySQL Workbench could see the database, but not Sequel Pro.

I found this Stack Overflow thread helpful:
https://stackoverflow.com/questions/51179516/sequel-pro-and-mysql-connection-failed

m.
